functests/1_performance/performance.go

```diff
@@ -252,17 +252,21 @@ var _ = Describe("[rfe_id:27368][performance]", func() {
 				ksoftirq_prio, err := strconv.Atoi(match[1])
 				Expect(err).ToNot(HaveOccurred())
 
-				rcuc_pid, err := nodes.ExecCommandOnNode([]string{"pgrep", "-f", "rcuc", "-n"}, &node)
-				Expect(err).ToNot(HaveOccurred())
-				Expect(rcuc_pid).ToNot(BeEmpty())
-				sched_tasks, err = nodes.ExecCommandOnNode([]string{"chrt", "-ap", rcuc_pid}, &node)
-				Expect(err).ToNot(HaveOccurred())
-				match = re.FindStringSubmatch(sched_tasks)
-				rcuc_prio, err := strconv.Atoi(match[1])
-				Expect(err).ToNot(HaveOccurred())
-
+				err = nodes.HasPreemptRTKernel(&node)
```

Comment:
let's avoid using `nodes.HasPreemptRTKernel(&node)` because it can return errors not only when the RT kernel is disabled
you can just check the profile

```
if profile.Spec.RealTimeKernel == nil ||
    profile.Spec.RealTimeKernel.Enabled == nil ||
    *profile.Spec.RealTimeKernel.Enabled != true {
	testlog.Warning("Skip checking rcu since RT kernel is disabled")
	return
}
```

Reply:
ok, thanks for letting me know, will change it.